skietbaan_fe/src/components/RegisterMember.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import { Container, Col, Form, FormGroup, Label, Input, Button, } from 'reactstrap';
+import '../components/RegisterMemberStyles.css';
+
+function validatePassword(str) {
+  var re = /(?=.*\d)(?=.*[a-z]).{6,}/;
+  return re.test(str);
+}
+
+ function validateEmail(email) {
+  var re = /.+@.+\..+/;
+  return re.test(String(email).toLowerCase());
+}
+
+ function validateUsername(username) {
+  var re = /[a-zA-Z]/;
+    return !re.test(String(username));
+}
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      usernameValue : "",
+      emailValue : "",
+      passwordValue:"",
+      invalidPassword:false,
+      invalidEmail:false,
+      invalidUsername: false,
+      validForm:false
+    }
+    this.RegisterMember = this.RegisterMember.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleChange({ target }) {
+    this.setState({
+      [target.name]: target.value,
+      invalidPassword:false,
+      invalidEmail: false,
+      invalidUsername: false
+    });
+
+    var Valid = true;
+    if(!validatePassword(this.state.passwordValue))
+    {
+        this.setState({
+          invalidPassword : true
+        });
+        Valid = false;
+    }
+    if(!validateEmail(this.state.emailValue))
+    {
+        this.setState({
+          invalidEmail : true
+        });
+        Valid = false;
+    }
+    if(validateUsername(this.state.usernameValue))
+    {
+        this.setState({
+          invalidUsername : true
+        });
+        Valid = false;
+    }
+    this.setState({
+      validForm: Valid
+    });
+  }
+
+  RegisterMember() {
+   if(this.state.validForm)
+   {
+    var RequestObject = {
+        "Username":this.state.usernameValue,
```

Comment:
Inconsistent Object declarations: 
Here you specify the keys with "" but earlier you did not. 
I suggest you use the method where you do not include the ""